data/core/units/drakes/Armageddon.cfg

```diff
@@ -20,6 +20,9 @@
     die_sound=drake-die.ogg
 
     {DEFENSE_ANIM "units/drakes/armageddon-defend-2.png" "units/drakes/armageddon-defend-1.png" {SOUND_LIST:DRAKE_HIT} }
+	[resistance]
```

Comment:
Please fix the indentation here and in the related changes.

Reply:
Huh, it was fine in Notepad ++. Will do.

Reply:
Looks good now. QQ